### PR TITLE
search-bar

### PR DESCRIPTION
**Pull request by Lav1004**

### CODE REVIEW

**Comment by Lav1004** [comment]
https://Lav1004.github.io/layout_search-bar-airbnb/
https://Lav1004.github.io/layout_search-bar-airbnb/report/html_report/

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf) format("truetype");
+  font-weight: 300;
+}
+
+.BigForm {
```
Comment (alexandra-protyanova) [inline src/style.css:13]: it isn't a good practice to name classes with a big letter

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]
Let us avoid horizontal scroll on the page and remove blue default outline from input

File: src/style.css (hunk 1 above)
Comment (Lav1004) [reply src/style.css]: corrected

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,45 @@
...
+          class="search-1"
+        >
+      </label>
+
+    </form>
+    <form 
+    action="/" 
+    method="post" 
+    class="smallForm" 
+    data-qa="small">
```
Comment (alexandra-protyanova) [inline src/index.html:39]:
```suggestion
    <form 
      action="/" 
      method="post" 
      class="smallForm" 
      data-qa="small"
    >
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,115 @@
+html {
+  width: 100%;
+  max-width: 100%;
+  overflow-x: hidden;
```
Comment (alexandra-protyanova) [inline src/style.css:4]: not a good solution

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+  box-sizing: border-box;
+  width: 19px;
+  height: 19px;
+}
+
+.search-1 {
+  padding-left: 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  /* box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1); */
```
Comment (alexandra-protyanova) [inline src/style.css:48]: delete comments

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+.search-bar {
+  position: relative;
+  width: 100%;
+}
+
+.iconBig {
+  position: absolute;
+  margin-left: 26px;
+  margin-top: 25px;
+  box-sizing: border-box;
```
Comment (alexandra-protyanova) [inline src/style.css:39]: move to `*` selector

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+.search-1:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.search-1::placeholder {
+  font-family: "Avenir", sans-serif;
```
Comment (alexandra-protyanova) [inline src/style.css:68]: move to `html` selector and use inherit where needed

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-style: normal;
```
Comment (alexandra-protyanova) [inline src/style.css:73]: it is a default property, remove it

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+  box-sizing: border-box;
+  width: 11px;
+  height: 11px;
+}
+
+.search-2 {
+  padding-left: 33px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  /* box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1); */
```
Comment (alexandra-protyanova) [inline src/style.css:89]: delete

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]
Horizontal scroll is not a good idea, need to fix. And let us remove default outline from input

File: src/style.css (hunk 8 above)
Comment (Lav1004) [reply src/style.css:89]: done

File: src/style.css (hunk 7 above)
Comment (Lav1004) [reply src/style.css:73]: done

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
1.image
2. Try to pass tests

**Pull request closed without merging**